nse_main.lua

```diff
@@ -409,6 +415,14 @@ do
       timeouts[self.host] = timeouts[self.host] or {};
       timeouts[self.host][self.co] = true;
     end
+    -- storing script's start time so as to account for script's timeout later
```

Comment:
If I am not wrong then `Thread:start()` would be called for every thread even if it is a worker thread hence, we can say that `timeout` will be assigned to every thread. One more slight change has been made in last commit and now at least http-slowloris times out after specified time limit.